alonzo/impl/src/Cardano/Ledger/Alonzo/Tx.hs

```diff
@@ -330,18 +342,68 @@ hashSD (w@(TxWitnessConstr (Memo (TxWitnessRaw _ _ scriptdata) _))) =
     else Just (ScriptDataHash (hashAnnotated scriptdata))
 -}
 
+-- ========================================================================
```

Comment:
Re the above comment - is this now fixed? I feel like this PR addresses some bits of it:

- Making it clear that the spec refers to two _different_ `ScriptDataHash`-es
- Renaming one of these to `WitnessLangDataHash`
- Allowing `WitnessLangDataHash` to be computed

However, are we still trying to compare hashes of two completely separate things? Or, looking at the spec, do we ever in fact need what is (now) called `ScriptDataHash` _at all_? It seems like the function `hashSD` is never in fact called. So perhaps we can completely drop `ScriptData` and reclaim the name for what's used here?

Appreciate that this is not necessarily the primary focus of this PR, but the above comment does draw attention to it, and it seems like fixing that would be a major bonus!